### DISCUSSION TITLE
GNOM configuration file

### DISCUSSION THREAD

**Discussion by skmkhdjh** [biosaxs-com/atsas-community#1964] [Primary data processing]
Hi,
I hope this message finds you well.

As I continue working on SAXS data analysis, I’ve been learning to use PRIMUS and GNOM but have encountered a few questions that I hoped you could help clarify. Specifically, I’m exploring the distance distribution function (P(r)) and Radius of Gyration calculations, and I’m trying to understand the differences between using PRIMUS for these analyses versus the GNOMqw window. Could you explain when one might be preferable over the other in terms of accuracy and configuration?

Additionally, I have some questions about GNOMqw's configuration options. I understand it can determine the ALPHA parameter, but I’m unsure if this parameter is critical for fine-tuning the P(r) distribution to achieve a reasonable or improved shape. Could you advise on the role of ALPHA here?

Lastly, as I’m new to the GNOM program, any simple explanations or tips would be immensely helpful.

Thank you very much for your time and guidance, which are invaluable as I work to build a foundation in these techniques.                                      

Warm regards, 
Dj

**Comment by franke-biosaxs** [comment]
You can use whichever you prefer. That said, be aware that `gnomqw` comes only with the _legacy_ package. The binary has not been updated for years. Any issues found and fixed in more recent releases are still present there. 

On details on ALPHA, please see the original publication. In short: ALPHA is a "smoothness" parameter of the p(r). If you set this value very high, the p(r) will be very smooth without (m)any oscillations - but the reconstructed scattering will likely not fit the data. Conversely, if you set this value low, i.e. close to zero, the fit to the experimental data will be great, but your p(r) will have many unreasonable oscillations. The middle ground is preferred. On the first run, gnom will provide an estimate of ALPHA.

Search this forum for questions and tips on how to deal with gnom and the p(r). There are plenty. HTH.
